RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -177,22 +180,22 @@ namespace lst {
                                                          float& zHiPointed,
                                                          float& sdlCut,
                                                          float& betaInCut,
-                                                         unsigned int& tripletIndex)
+                                                         const unsigned int tripletIndex)
 #else
   ALPAKA_FN_ACC ALPAKA_FN_INLINE void addTripletToMemory(struct lst::Modules& modulesInGPU,
                                                          struct lst::MiniDoublets& mdsInGPU,
                                                          struct lst::Segments& segmentsInGPU,
                                                          struct lst::Triplets& tripletsInGPU,
-                                                         unsigned int& innerSegmentIndex,
-                                                         unsigned int& outerSegmentIndex,
-                                                         uint16_t& innerInnerLowerModuleIndex,
-                                                         uint16_t& middleLowerModuleIndex,
-                                                         uint16_t& outerOuterLowerModuleIndex,
+                                                         const unsigned int innerSegmentIndex,
+                                                         const unsigned int outerSegmentIndex,
+                                                         const uint16_t innerInnerLowerModuleIndex,
+                                                         const uint16_t middleLowerModuleIndex,
+                                                         const uint16_t outerOuterLowerModuleIndex,
                                                          float& betaIn,
                                                          float& circleRadius,
                                                          float& circleCenterX,
                                                          float& circleCenterY,
```

Comment:
probably also to be by value; 
please check other `addXYZToMemory` methods; they are not supposed to compute anything per object, just to pass the input data to SoAs